monitors/BalanceMonitor.js

```diff
@@ -13,7 +13,7 @@ class BalanceMonitor {
   //   syntheticThreshold: x2,
   //   etherThreshold: x3 },
   // ...]
-  constructor(logger, tokenBalanceClient, botsToMonitor) {
+  constructor(logger, tokenBalanceClient, botsToMonitor, empProps) {
```

Comment:
nit: could you document what's in this object above the constructor? Here and wherever else it's passed around?

Reply:
yup. this commenting needs big improvement. I intentionally did not comment on this interface because I want to go through all bots and make sure they are consistent. right now there is a bit of a deviation in standards. What fields are commented and the formatting thereof. for example this BalanceMonitor uses `//`, CRMonitor uses `*` and the ContractMonitor has no comments on the constructor.

I think it would be best to leave this for a separate PR that implements this change across all bots and closes issue [1446](https://github.com/UMAprotocol/protocol/issues/1446).

Reply:
+1

Reply:
SGTM